user_install.sh

```diff
@@ -358,8 +364,18 @@ fi
 # Write the MIL runcom file for sourcing all of the required project configurations
 echo "# This file is created by the install script to source all of the configurations" > $MILRC_FILE
 echo "# needed to work on the installed projects. Do not edit this file manually! Your" >> $MILRC_FILE
-echo "# changes will be overwritten the next time the install script is run. Please use" >> $MILRC_FILE
-echo "# the script to make changes." >> $MILRC_FILE
+echo "# changes will be overwritten the next time the install script is run. Please" >> $MILRC_FILE
+echo "# use the script to make changes." >> $MILRC_FILE
+
+# Add CUDA programs to the path if the Nvidia repository was used to install it
+if ($INSTALL_CUDA); then
+	if [ "$REQUIRED_OS_ID" = "Ubuntu" ]; then
+		echo "" >> $MILRC_FILE
+		echo "# Adds CUDA programs and libraries to the shell's path" >> $MILRC_FILE
+		echo "export PATH=$PATH:/usr/local/cuda/bin" >> $MILRC_FILE
```

Comment:
That's a good point, but the script uses the -qq flag on apt, which means the link will be created.